rest/handler.go

```diff
@@ -557,6 +559,20 @@ func (h *handler) logRequestLine() {
 	base.InfofCtx(h.ctx(), base.KeyHTTP, "%s %s%s%s", h.rq.Method, base.SanitizeRequestURL(h.rq, &queryValues), proto, h.formattedEffectiveUserName())
 }
 
+// logBytesWritten only logs database level bytes written over the public API
+func (h *handler) logBytesWritten() {
+	if h.db == nil {
+		return
+	}
+	if h.privs != publicPrivs && h.privs != regularPrivs {
+		return
+	}
+	if h.isBlipSync() {
```

Comment:
How are we handling bytes associated with streaming non-blip _changes requests (0feed=continuous or feed=websocket)?

Reply:
These are actually not done with `http.ResponseWriter` so potentially I don't even have to skip this here, I'm not sure what is more clear.